src/components/basic/film/ppFilm.tsx

```diff
@@ -0,0 +1,43 @@
+// *Styles*
+import "./ppFilm.scss";
+
+import Chip from "../chip/chip";
+
+interface Props {
+  textColour?: string;
+}
+
+const fakeData = {
+  title: "The Avengers",
+  poster: "icons/poster.jpeg",
+  description:
+    "Nick Fury is compelled to launch the Avengers Initiative when Loki poses a threat to planet Earth. His squad of superheroes put their mindas together to accomplish the task.",
+  releaseYear: "2012",
+  genres: ["Action", "Superhero"],
+};
+
+const PPFilm = ({ textColour = "primary" }: Props) => {
+  return (
+    <div className="pp-film-container">
+      <img
+        src={fakeData.poster}
+        alt="movie poster"
```

Comment:
If an image is for decorative purposes only (like this one) it's better to leave the `alt` as blank as this isn't a useful thing for someone with a screen reader to hear https://www.w3.org/WAI/tutorials/images/decorative/